workflow/utils/paths.py

```diff
@@ -7,18 +7,64 @@ def get_ephys_root_data_dir():
     return pathlib.Path(data_dir) if data_dir else None
 
 
-def get_processed_root_data_dir():
+def get_ephys_processed_root_data_dir():
     data_dir = dj.config.get('custom', {}).get('ephys_processed_data_dir', None)
     return pathlib.Path(data_dir) if data_dir else None
 
 
-def get_session_directory(session_key: dict) -> str:
+def get_ephys_session_directory(session_key: dict) -> str:
```

Comment:
Would this rename break the code somewhere in the workflow?

Reply:
I think it should work since everywhere we use it we import it right? So as long as all the imports are correctly changed it should work. But I will double check to see if we do use it elsewhere

Reply:
If it is used by the element (when activating the element), the element may expect the name of the function to be `get_session_directory` instead of `get_ephys_session_directory`
I'm don't remember which element uses this, if any

Reply:
Ah ok, that does make sense, I'll look further into the activation and see if it will hold for all of these elements